pkg/apis/autoheal/v1alpha1/doc.go

```diff
@@ -16,5 +16,5 @@ limitations under the License.
 
 // +k8s:deepcopy-gen=package
 
-// +groupName=monitoring.openshift.io
-package v1alpha1 // import "github.com/openshift/autoheal/pkg/apis/monitoring/v1alpha1"
+// +groupName=autoheal.openshift.io
+package v1alpha1 // import "github.com/openshift/autoheal/pkg/apis/autoheal/v1alpha1"
```

Comment:
Since you are changing the api, don't you want to change the package version?
something like `v1alpha2`?

Reply:
Done in the next version of the pull request.